README.md

```diff
@@ -1,116 +1,209 @@
-[![Snyk logo](https://snyk.io/style/asset/logo/snyk-print.svg)](https://snyk.io)
+# Snyk Security
```

Comment:
We do use "Snyk Security Scanner" in some places as well, including the name of this repo. I like the Scanner there as well, because it makes it more obvious what is it doing - what do you think about the canonical name here?

Reply:
I've kept it the same as the current marketplace page for now.

https://plugins.jenkins.io/snyk-security-scanner/

We can consider making the name consistent as a separate PR as it'll touch a few places. It's not called "Scanner" in the plugin manager page (via pom.xml I assume), nor in the plugin configuration for Freestyle ("Snyk Security") and Pipelines (`snykSecurity`).